pkg/app/web/src/components/applications-page/index.tsx

```diff
@@ -176,7 +176,7 @@ export const ApplicationIndexPage: FC = () => {
         ModalProps={{ disableBackdropClick: isAdding }}
       >
         {addedApplicationId && (
-          <DeploymentConfigForm onSkip={handleCloseTemplateForm} />
+          <ApplicationAddedView onClose={handleCloseTemplateForm} />
```

Comment:
nits, better to add a TODO note to remove this `handleCloseTemplateForm`

Reply:
Sure. Added!